gui/system/views.py

```diff
@@ -1009,8 +1010,10 @@ def debug_download(request):
     gc = GlobalConfiguration.objects.all().order_by('-id')[0]
 
     _n = notifier()
-    if not _n.is_freenas() and _n.failover_licensed():
-        debug_file = '%s/debug.tar' % direc
+    dual_node_debug_file = debug_file = '{}/debug.tar'.format(direc)
+
+    if not _n.is_freenas() and _n.failover_licensed() and os.path.exists(dual_node_debug_file):
+        debug_file = dual_node_debug_file
```

Comment:
Just a suggestion but I think the `dual_node_debug_file` var is unnecessary.

May I suggest this instead:
```
debug_file = f'{direc}/debug.tar'

if not _n.is_freenas() and _n.failover_licensed() and os.path.exists(debug_file):
    extension = 'tar'
...
```